docs/source/schema-design/federated-schemas/federation.mdx

```diff
@@ -1,27 +1,19 @@
 ---
 title: Introduction to Apollo Federation
-subtitle: Learn how federation combines your GraphQL APIs into a unified supergraph
-description: Learn how Apollo Federation can help you declaratively combine your services into a unified, federated GraphQL API using a microservices architecture.
+subtitle: Learn how federation orchestrates your APIs into a unified supergraph
+description: Learn how Apollo Federation helps you declaratively orchestrate your APIs and services into a unified, federated GraphQL API using a microservices architecture.
 redirectFrom:
     - /federation
 ---
 
-Apollo Federation lets you declaratively combine multiple APIs into a single, federated graph. This federated graph enables clients to interact with your APIs through a single request.
+Apollo Federation enables you to declaratively combine multiple APIs into a single federated graph. A client makes a single request to a federated GraphQL API, and federation orchestrates calls to multiple APIs to produce a single response.  
 
 A client makes a request to the federated GraphQL API's single entry point called the _router_. The router intelligently orchestrates and distributes the request across your APIs and returns a unified response. For a client, the request and response cycle of querying the router looks the same as querying any GraphQL server.
 
 <img src='../../images/federation.svg' class="dark:hidden"/>
 <img src='../../images/federation-dark.svg' class="hidden dark:block"/>
```

Comment:
Should we replace this diagram with one that includes connectors?

Reply:
good call, we can update in follow-up PR